pydicom/util/testing/_write_stds.py

```diff
@@ -13,27 +14,27 @@
     b"00 00 00 00 "     # length 0
     b"06 30 39 00 "     # (3006, 0039) ROI Contour Sequence
     b"5a 00 00 00 "     # length 90
-        b"fe ff 00 e0 "     # (fffe, e000) Item Tag
-        b"52 00 00 00 "     # length 82
-            b"06 30 40 00 "     # (3006, 0040)  Contour Sequence
-            b"4a 00 00 00 "     # length 74
-                b"fe ff 00 e0 "     # (fffe, e000) Item Tag
-                b"1a 00 00 00 "     # length 26
-                    b"06 30 48 00 "     # (3006, 0048) Contour Number
-                    b"02 00 00 00 "     # length 2
-                    b"31 20 "           # "1 "
-                    b"06 30 50 00 "     # (3006, 0050) Contour Data
-                    b"08 00 00 00 "     # length 8
-                    b"32 5c 34 5c 38 5c 31 36 "  # "2\4\8\16"
-                b"fe ff 00 e0 "     # (fffe, e000) Item Tag
-                b"20 00 00 00 "     # length 32
-                    b"06 30 48 00 "     # (3006, 0048) Contour Number
-                    b"02 00 00 00 "     # length 2
-                    b"32 20 "           # "2 "
-                    b"06 30 50 00 "     # (3006, 0050) Contour Data
-                    b"0e 00 00 00 "     # length 14
-                    b"33 32 5c 36 34 5c 31 32 38 5c 31 39 36 20 "
-                                                            # "32\64\128\196 "
+    b"fe ff 00 e0 "     # (fffe, e000) Item Tag
+    b"52 00 00 00 "     # length 82
+    b"06 30 40 00 "     # (3006, 0040)  Contour Sequence
+    b"4a 00 00 00 "     # length 74
+    b"fe ff 00 e0 "     # (fffe, e000) Item Tag
+    b"1a 00 00 00 "     # length 26
+    b"06 30 48 00 "     # (3006, 0048) Contour Number
+    b"02 00 00 00 "     # length 2
+    b"31 20 "           # "1 "
+    b"06 30 50 00 "     # (3006, 0050) Contour Data
+    b"08 00 00 00 "     # length 8
+    b"32 5c 34 5c 38 5c 31 36 "  # "2\4\8\16"
+    b"fe ff 00 e0 "     # (fffe, e000) Item Tag
+    b"20 00 00 00 "     # length 32
+    b"06 30 48 00 "     # (3006, 0048) Contour Number
+    b"02 00 00 00 "     # length 2
+    b"32 20 "           # "2 "
+    b"06 30 50 00 "     # (3006, 0050) Contour Data
+    b"0e 00 00 00 "     # length 14
+    b"33 32 5c 36 34 5c 31 32 38 5c 31 39 36 20 "
+                        # "32\64\128\196 "
 )
```

Comment:
Yeah I intuited that - what we could do is go back to the original formatting, but then we would need a `# noqa` at the end of each line. Is one more preferable? Are people needing to read this regularly?